tex/chapters/MachineLearning.tex

```diff
@@ -1,3 +1,267 @@
 %!TEX root = ../main.tex
 
 \chapter{Machine Learning}\label{cha:machinelearning}
+
+In this chapter a new approach to identifying strategies that are the same will be presented.
+A new type of tournament will be presented, and the summary of the results of this tournament will then be used to train a Machine Learning model.
+The model is then able to predict whether new strategies are the same as ones already implemented within Axelrod-Python.
+
+
+
+\section{Ashlock Tournament}\label{sec:ashlock_tourn}
+
+In an Ashlock tournament, all strategies entered play against a probe strategy for a range of parameters, and do not play each other.
```

Comment:
Re word / re structure this, it's the same tournament as used in Ashlock fingerprinting so move this to there and point towards it.